client/js/actions/snippet-actions.js

```diff
@@ -11,8 +12,24 @@ class SnippetActions {
 
   getAll() {
     axios.get('/snippets')
-      .then(res => this.dispatch({ok: true, snippets: res.data}))
-      .catch(err => this.dispatch({ok: false, error: err}));
+      .then(res => {
+        FlashMessages.pushMessage({ content: 'Here are all of the snippets!' });
```

Comment:
I think actions should only dispatch view events to change store and I see displaying as flash message as view logic. Also now this actions is used by one view but later we could find a situation for using it without flash.